plaso/containers/manager.py

```diff
@@ -33,15 +33,24 @@ def DeregisterAttributeContainer(cls, attribute_container_class):
 
   @classmethod
   def GetAttributeContainer(cls, container_type):
-    """Retrieves the attribute container for a specific container type.
+    """Retrieves an instance of a specific attribute container type.
```

Comment:
-Retrieves +Creates 

It's a brand new container being created, not retrieving an existing class any more - maybe even rename the method to CreateAttributeContainer

Reply:
SG